tools/ngsderive.wdl

```diff
@@ -190,16 +268,38 @@ END
     runtime {
         memory: "~{memory_gb} GB"
         disk: "~{disk_size_gb} GB"
-        docker: 'quay.io/biocontainers/ngsderive:2.4.0--pyhdfd78af_0'
+        docker: 'quay.io/biocontainers/ngsderive:3.3.2--pyhdfd78af_0'
         maxRetries: max_retries
     }
 }
 
 task junction_annotation {
+    meta {
+        description: "Annotates junctions found in an RNA-Seq BAM as known, novel, or partially novel"
```

Comment:
Do you think it is worth defining `known`, `novel`, and `partially novel` here or in a `help` text?

Reply:
To add to that, I think the definitions are missing in the `ngsderive` docs. It might be better there. RSeQC states: 

> 
> Annotated (known): The junction is part of the gene model. Both splice sites, 5’ splice site (5’SS) and 3’splice site (3’SS) are annotated by reference gene model.
> 
> Complete_novel: Both 5’SS and 3’SS are novel.
> 
> Partial_novel: One of the splice site (5’SS or 3’SS) is novel, and the other splice site is annotated

Reply:
I'll add to the ngsderive docs, and link to those docs here using the `external_help` key 👍

Reply:
Definitions have been added to the ngsderive v4 PR. [specific commit with those definitions](https://github.com/stjudecloud/ngsderive/pull/139/commits/625210e30ad2aea2b3ca8bcf0185db223a966981)